ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/DeployForm/DeployFormFields/DeployFormFields.tsx

```diff
@@ -142,7 +155,11 @@ export const DeployFormFields = (): JSX.Element => {
               }}
               type="checkbox"
             />
-            <p className="p-form-help-text" style={{ paddingLeft: "2rem" }}>
+            <p
+              id={deployVmHostHelpText}
+              className="p-form-help-text"
+              style={{ paddingLeft: "2rem" }}
```

Comment:
We have `u-nudge-right` utility classes so you shouldn't need to inline style this anymore

Reply:
I didn't add this, but can update 👍

Reply:
Seems like it was missing a `is-tick-element` className - that's what makes it align with the checkbox properly. Updated across all inputs on this form.

Reply:
Opened a PR fixing this in react-components https://github.com/canonical-web-and-design/react-components/pull/751